examples/src/keyrings/aws_kms_mrk_multi_keyring_example.py

```diff
@@ -0,0 +1,175 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the KMS MRK Multi Keyring
+
+The AWS Key Management Service (AWS KMS) MRK keyring interacts with AWS KMS to
+create, encrypt, and decrypt data keys with multi-region AWS KMS keys (MRKs).
+This example creates a KMS MRK Multi Keyring using an mrk_key_id (generator) and
+a kms_key_id, and then encrypts a custom input EXAMPLE_DATA with an encryption context.
```

Comment:
What does this combination of generator/kms key ID do? What does providing the extra KMS key ID get us? Why might we do this? 

You can be really brief here since you explain this further down.